understanding/understanding-techniques.html

```diff
@@ -490,38 +439,30 @@ <h3>Support Notes Change Over Time</h3>
-                     in Understanding WCAG 2.1 and to specific techniques.
+                  <p><a href="https://www.w3.org/WAI/WCAG22/Techniques/">Techniques for WCAG 2.2</a> は単体の文書として用いられることを意図していない。代わりに、コンテンツ制作者は、WCAG 2 の達成基準を参照するために 
+                     <a href="https://www.w3.org/WAI/WCAG22/quickref/">How to Meet WCAG 2.2: A customizable quick reference</a> を用い、そこからリンクをたどって WCAG 2.2 解説書の特定の項及び特定のテクニックを参照することが想定している。
                   </p>
                   			
                   <section id="alternatives-must-meet-success-criteria">
                      				
-                     <h3>Alternatives must meet success criteria</h3>
+                     <h3>代替版は達成基準を満たしていなければならない</h3>
                      				
-                     <p>Some techniques describe how to provide alternate ways for users to get content. For
-                        example, <a href="https://w3c.github.io/wcag/techniques/general/G73">G73: Providing a long description in another location with a link to it that is immediately
-                           adjacent to the non-text content</a> mentions a transcript as an alternative for an audio file. Some alternatives must
-                        also conform to WCAG. For example, the transcript itself must meet all relevant success
-                        criteria. 
+                     <p>いくつかの達成基準では、ユーザがコンテンツを得るための代替方法をどのように提供するかについて述べている。例えば、<a href="https://w3c.github.io/wcag/techniques/general/G73">G73: Providing a long description in another location with a link to it that is immediately
+                           adjacent to the non-text content</a> では音声ファイルの代替としてトランスクリプトについて言及している。いくつかの代替はまた WCAG 2 に適合していなければならない。例えばそのトランスクリプト自体は、関連するすべての達成基準を満たしていなければならない。
                      </p>
                      			
                   </section>
                   			
                   <section id="example-code">
                      				
-                     <h3>Example Code</h3>
+                     <h3>コード例</h3>
                      				
-                     <p>The code examples in the techniques are intended to demonstrate only the specific
-                        point discussed in the technique. They might not demonstrate best practice for other
-                        aspects of accessibility, usability, or coding not related to the technique. They
-                        are not intended to be copied and used as the basis for developing web content.
+                     <p>テクニック集にあるコード例は、そのテクニックにおいて議論された特定の点についてのみ明示することを意図している。これらは、そのテクニックと無関係なアクセシビリティ、ユーザビリティ、またはコーディングのその他の側面に対する最良の実例を示しているわけではない。これらはウェブコンテンツ開発の基礎としてコピーして用いられることを意図していない。
                      </p>
                      				
-                     <p>Many techniques point to "working examples" that are more robust and may be appropriate
-                        for copying and integrating into web content.
+                     <p>多くのテクニックではより堅牢で、ウェブコンテンツにコピーしたり統合するのに適した "機能する例" を挙げている。
```

Comment:
"working examples" を "機能する例" としていますが、「機能する」と言われると「役に立つ」のようなニュアンスにも読めます。ここでは単に、ブラウザでアクセスした際に実際に「動く」という意味ですので、動作する例、実働例というくらいではないでしょうか。

```suggestion
                     <p>多くのテクニックではより堅牢で、ウェブコンテンツにコピーしたり統合するのに適した "動作する例" を挙げている。
```

Reply:
取り込みました。